packages/core-types/addon/components/field-editors/integer-editor.js

```diff
@@ -3,4 +3,16 @@ import layout from '../../templates/components/field-editors/integer-editor';
 
 export default Component.extend({
   layout,
+
+  actions: {
+    update(value) {
+      let content = this.get('content');
+      let field = this.get('field');
+      if (value == null || value === '') {
```

Comment:
why not `!value`? I'm assuming `value` can only be something nullish or a string?

Reply:
I restored this piece of code from the previous version of the integer editor where it was written that way. I don't think it makes a difference, though.